monkey/monkey_island/cc/next_ui/src/_components/drawer/DrawerMonkeyDrawer.cy.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import MonkeyDrawer from './Drawer';
+
+describe('<MonkeyDrawer />', () => {
+    it('renders', () => {
+        // see: https://on.cypress.io/mounting-react
+        cy.mount(<MonkeyDrawer />);
```

Comment:
So this just tests if it mounts? Since these tests aren't doing much it's probably better to delete them and introduce a test once we have something we want to test.